src/style.css

```
@@ -0,0 +1,78 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Arial, sans-serif;
+}
+
+.blocks {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  font-size: 100px;
+  height: 300px;
+  width: 33.333%;
```

```suggestion
  width: 33.33%;
```
двух знаков достаточно